gradle/mockito-core/inline-mock.gradle

```diff
@@ -0,0 +1,13 @@
+task copyMockMethodDispatcher(type: Copy) {
+    from "${buildDir}/classes/main/org/mockito/internal/creation/bytebuddy"
+    into "${buildDir}/classes/main/org/mockito/internal/creation/bytebuddy"
+    include "MockMethodDispatcher.class"
+    rename { String fileName ->
+        fileName.replace('.class', '.raw')
+    }
+}
+classes.dependsOn(copyMockMethodDispatcher)
```

Comment:
In order for this to work reliably, we need to also set one more task dependency on compileJava. Let me try to update the code in the branch...